doc/2/controllers/models/update-asset/index.md

```diff
@@ -48,7 +48,12 @@ Method: PUT
                 description: string;
               };
             };
-            readOnly?: boolean;
+            definition?: {
+              isReadOnly?: boolean;
```

Comment:
I would rather call it readonly: boolean for simplicity

Reply:
I'm ok with the changing but putting 'is', 'has', etc... for boolean is not a convention ?

Reply:
I agree with Ricky, the `is` isn't necessary, `readOnly` is enough